pkgs/development/python-modules/spyder/default.nix

```diff
@@ -41,24 +37,21 @@
 , textdistance
 , three-merge
 , watchdog
+, yapf
 }:
 
 buildPythonPackage rec {
   pname = "spyder";
-  version = "5.4.2";
+  version = "5.4.4";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-L8zgT7M7N+P5/9FQgf6ab7afUQXC1afzwUjAp6yKxC8=";
+    hash = "sha256-+AUpKPjRU1it5HvYutCY/N5fH3okdQjJaYMMLPawwKI=";
   };
 
-  patches = [
-    ./dont-clear-pythonpath.patch
```

Comment:
Did you forget to remove this patch from git?

Reply:
Nice catch, thanks !